ContributedExamples/Hovmoller_Temperature_Depth.ipynb

```diff
@@ -6,6 +6,559 @@
    "source": [
```

Comment:
<p>Why do we need this declaration of who wrote what and when.</p><p> </p><p>Felipe suggest what you think should stay and delete everything else. Please don't include names of who did what.... Once in the recipes there is no ownership in the notebooks or on the code lines -- code belongs to the community. :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>